stl/inc/chrono

```diff
@@ -609,8 +610,22 @@ namespace chrono {
         static constexpr bool is_steady = true;
 
         _NODISCARD static time_point now() noexcept { // get current time
-            const long long _Freq = _Query_perf_frequency(); // doesn't change after system boot
-            const long long _Ctr  = _Query_perf_counter();
+#if (defined(_M_ARM) || defined(_M_IX86) || defined(_M_ARM64) || defined(_M_X64)) && !defined(_M_CEE_PURE)
+            // ad-hoc <atomic> load/compare_exchange_strong with std::memory_order_relaxed
+            // avoid dependency on real <atomic>
+            static volatile long long _Freq_cached = -1; // doesn't change after system boot

```

Comment:
You need to make sure this has 64 bit alignment on 32 bit or `__iso_volatile_load64` won't be atomic.

Reply:
Good point. Maybe both warning I"m trying to suppress and __iso_atomic_store ICE are caused by this.

Reply:
But alignof(long long) == sizeof(long long) , which is 8.

So putting `alignas` looks superfluous, ans `static_assert` on `alignof` looks paranoid.

And even with `/Zp1` this variable is on its natural alignment, as it is not a member of any structure. 

So I'm not sure what should be done.

Reply:
> `alignof(long long) == sizeof(long long)`

It isn't on the stack on x86 but maybe we are OK here because it isn't an automatic variable.